tcms/report/data.py

```diff
@@ -37,7 +36,11 @@
 
 
 def models_to_pks(models):
-    return [model.pk for model in models]
```

Comment:
FYI you can leave simple list comprehensions like this untouched and disable the warning for them.